### PR TITLE
Fixing Regression for chrome stale elements

### PR DESCRIPTION
**Pull request by abjerstedt**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [X] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/webdriver/src/utils.js [hunk 1]
```diff
@@ -316,6 +316,10 @@ export function getErrorFromResponseBody (body) {
 export class CustomRequestError extends Error {
     constructor(body) {
         super(body.value.message || body.value.class || 'unknown error')
-        if (body.value.error) this.name = body.value.error
+        if (body.value.error) {
+            this.name = body.value.error
+        } else if (body.value.message && body.value.message.includes('stale element reference')) {
```
Comment (abjerstedt) [inline packages/webdriver/src/utils.js]: It looks like chrome isn't passing the error property as expected, so we'll sniff it, and hardcode the name if matched

Comment (lieutenantken) [reply packages/webdriver/src/utils.js:321]: Can we add the element/selector properties to the error message. Just how do you expect users to diagnose 👍 
WARN webdriver: Request encountered a stale element - terminating request

**Comment by lieutenantken** [comment]
Can we add the element/selector properties to the error message. Just how do you expect users to diagnose 👍
WARN webdriver: Request encountered a stale element - terminating request

**Comment by christian-bromann** [comment]
@lieutenantken makes sense, feel free to propose a PR

**Pull request merged**